pkg/webui/containers/side-bar/navigation/app-side-navigation.js

```diff
@@ -0,0 +1,133 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useCallback, useContext } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useSelector } from 'react-redux'
+import { defineMessages } from 'react-intl'
+
+import SideNavigation from '@ttn-lw/components/navigation/side-v2'
+import DedicatedEntity from '@ttn-lw/components/dedicated-entity'
+
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import {
+  selectSelectedApplication,
+  selectSelectedApplicationId,
+} from '@console/store/selectors/applications'
+
+import SidebarContext from '../context'
+
+const m = defineMessages({
+  buttonMessage: 'Back to Applications list',
+})
+
+const AppSideNavigation = () => {
+  const app = useSelector(selectSelectedApplication)
+  const appId = useSelector(selectSelectedApplicationId)
+  const { isMinimized, setLayer } = useContext(SidebarContext)
+  const navigate = useNavigate()
+
+  const entityId = app ? app.name ?? appId : appId
+
+  const handleBackClick = useCallback(() => {
+    const path = '/applications'
+    navigate(path)
+    setLayer(path)
+  }, [navigate, setLayer])
+
+  return (
+    <>
+      <SideNavigation>
+        {!isMinimized && (
+          <DedicatedEntity
+            label={entityId}
+            buttonMessage={m.buttonMessage}
+            icon="arrow_left"
+            className="mt-cs-xs mb-cs-m"
+            onClick={handleBackClick}
+          />
+        )}
+        <SideNavigation.Item
+          title={sharedMessages.appOverview}
+          path={`applications/${appId}`}
+          icon="group"
+          exact
+        />
+        <SideNavigation.Item
+          title={sharedMessages.devices}
+          path={`applications/${appId}/devices`}
+          icon="device"
+        />
+        <SideNavigation.Item
+          title={sharedMessages.liveData}
+          path={`applications/${appId}/data`}
+          icon="list_alt"
+        />
+        {/* <SideNavigation.Item title={'Network Information Center'} path="/noc" icon="ssid_chart" /> */}
```

Comment:
Needs to be uncommented (and attached to feature checks).

Reply:
Isn't the NOC link meant to just be on enterprise?

Reply:
Ah, you're right. Let's remove the comment then.